ghost/core/test/unit/server/lib/request-external.test.js

```diff
@@ -300,7 +303,9 @@ describe('External Request', function () {
             }, (err) => {
                 requestMock.isDone().should.be.true();
                 should.exist(err);
-                err.statusMessage.should.be.equal('Not Found');
+                // console.log(err);
+                // TODO: no error code or status message
+                err.code.should.be.equal(`ERR_NON_2XX_3XX_RESPONSE`);
```

Comment:
According to the documentation (https://github.com/sindresorhus/got/tree/v11#gothttperror), we should be able to access the status etc via `response`

```suggestion
err.response.statusMessage.should.be.equal('Not Found');
```

Does that work? I so, I would clean this up for the other tests too and remove the comments.

Reply:
Awesome, yes. Not sure why I wasn't seeing that last week.